constants/menuLinks.ts

```diff
@@ -175,7 +175,7 @@ const FOOTER_ELEMENTS: FooterStructure = [
       QISKIT_ELEMENTS.ignis
     ]
   },
-  // Elements in the third footer column
+  // Elements in the second footer column
```

Comment:
Non-blocking: instead of a comment per entry, you can clarify this before the `const`: `// Each entry represents a column`.